.circleci/config.yml

```diff
@@ -20,7 +20,7 @@ workflows:
           name: build_docker_and_test
           cmd: |
             docker build -t $CIRCLE_PROJECT_REPONAME:latest .
-            docker run $CIRCLE_PROJECT_REPONAME:latest codacy_pylint_test.py
+            docker run --rm $CIRCLE_PROJECT_REPONAME:latest codacy_pylint_test.py
```

Comment:
Does it make a difference when running in CI?

Reply:
I don't think so.. They probably kill everything either way.